.github/workflows/remark-lint-problem-matcher.json

```diff
@@ -0,0 +1,23 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "remark-lint",
+      "pattern": [
+        {
+          // Filename may be decorated by ANSI escape sequences
```

Comment:
JSON doesn't support comments, although if this works anyway, hey, cool.

Reply:
It works but I can get rid of the comment.